softdevice_controller/README.rst

```diff
@@ -21,25 +21,23 @@ SoC support:
 +============+========+========+========+
 | nRF52805   | Rev 1  |        |        |
 +------------+--------+--------+--------+
-| nRF52810   | Rev 1  |        |        |
+| nRF52810   | Rev 1  | Rev 1  |        |
```

Comment:
@jostn , can you verify that this is correct?
Do we need to update the allowed configurations in Kconfig?

Reply:
Verified that this is as expected. Pushed https://github.com/nrfconnect/sdk-nrfxlib/pull/265/commits/e06d8d3dda452d808817531ea40866a97c158c3d to update the library selection limitations